src/test/app/claims/responseModelConverter.ts

```diff
@@ -221,7 +221,7 @@ describe('ResponseModelConverter', () => {
         .to.deep.equal(convertObjectLiteralToJSON(responseData))
     })
 
-    it('should convert partial admission paid by instalments with complete SoM', () => {
+    it.skip('should convert partial admission paid by instalments with complete SoM', () => {
```

Comment:
I can understand why you skipped these tests from your commit comments but i usually draft over-ride claimant-response data in specific test because it gives you flexibility to change as per your specific test. Maybe you can try if its easy...or i can pick it up later

Reply:
After discussing with @srineelesh , i think its best to leave the tests as skipped for now until later when completionDate and paymentLength are added to PaymentPlan module.